File: 1293-three-consecutive-odds/1293-three-consecutive-odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int x = 0;
        int y = 1;
        int z = 2;
        int n = arr.length;
        while(x<n & y<n & z<n){
            if(arr[x]%2 != 0){
                if(arr[y]%2 != 0){
                    if(arr[z]%2!=0){
                        return true;
                    }
                }
            }
            x++;
            y++;
            z++;
        }
        return false;  
    }
}

File: 1293-three-consecutive-odds/1293-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        if len(arr) == 3:
            if arr[0]%2!=0 and arr[1]%2!=0 and arr[2] % 2 != 0:
                return True
        else:
            for i in range(0,len(arr)-2):
                if arr[i]%2!=0 and arr[i+1]%2!=0 and arr[i+2] % 2 != 0:
                    return True
        